scss/components/_accordion.scss

```diff
@@ -65,24 +67,30 @@ $accordion-content-padding: 1rem !default;
 }
 
 /// Adds styles for the title of an accordion item. Apply this to the link within an accordion item.
-@mixin accordion-title {
+@mixin accordion-title (
+  $padding: $accordion-item-padding,
+  $font-size: $accordion-title-font-size,
+  $color: $accordion-item-color,
+  $border: $accordion-content-border,
+  $background-hover: $accordion-item-background-hover
+) {
   display: block;
-  padding: $accordion-item-padding;
+  padding: $padding;
   line-height: 1;
-  font-size: $accordion-title-font-size;
-  color: $accordion-item-color;
+  font-size: $font-size;
+  color: $color;
```

Comment:
Oh, and I'm only using alt colors for component that create classes with an `@each` loop for the `$foundation-palette`. Components that do not use a `$foundation-palette` loop do not need an alt color.